version1/02 - Object types.md

```diff
@@ -90,863 +94,639 @@ The following notation is used in the tables below:
-|           | ref       | refNasjonalIdentifikator         |          |             |                    |  |      |      |   x   |      |  x   |        |                                                                                                                                                               |
-|           | ref       | refKryssreferanseTilMappe        |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktMappe** that the current **AbstraktRegistrering** cross references                                                                        |
-|           | ref       | refKryssreferanseFraMappe        |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktMappe** that cross reference the current **AbstraktRegistrering**                                                                         |
-|           | ref       | refKryssreferanseTilRegistrering |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that the current **AbstraktRegistrering** cross references                                                                 |
-|           | ref       | refKryssreferanseFraRegistrering |          |             |                    |  |      |      |   x   |      |  x   |   x    | refers to **AbstraktRegistrering** that cross reference the current **AbstraktRegistrering**                                                                  |
-
-### Journalpost
-
-| Field no. | Type      | Field                             | Not null | Has default |    Code list       |  | Save | View | Query | Sort | Link | Unlink | Comment                                                                                                                                                                                                                                                                                                                                                                                                         |
+Note: **EFO** required if finalized
+
+Note: **EJO** required if journaled
+| Field no. | Type      | Field                            | Not null |   Default   | Code list          |  | Save | View | Query | Sort | Link | Unlink | Comment |
+|:----------|:----------|:---------------------------------|:--------:|:-----------:|:-------------------|:-|:----:|:----:|:-----:|:----:|:----:|:------:|:--------|
+| M110      | date      | offentlighetsvurdertDato         |          |             |                    |  |  V   |  V   |   V   |  V   |      |        |         |
+
+
+### Registry Entry ([Basisregistrering](#basisregistrering-finalizable-object))
```

Comment:
You have the Registry Entry section here, which is not complete, and then a section for the Journalpost, which is complete. I think this Registry Entry section is redundant?